wp-content/plugins/core/src/Router/Routes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tribe\Project\Router;
+
+/**
+ * Class Routes
+ *
+ * @package Tribe\Project\Router
+ */
+class Routes {
+
+	/**
+	 * @param Router $router
+	 */
+	public function add_routes( Router $router ) {
```

Comment:
Does this allow passing in dependencies to the Controller? What would that look like? Some Controllers do a bit more "Model' logic that i feel could be moved into those classes.

Reply:
Excellent question - and I think that's a very good point. One of the big issues I had with the existing system was the inability to inject dependencies to the Controllers, and I absolutely can see how that would be a good idea here. 

It'll take a bit of refactoring, but since we're already defining the Controllers we want to use in the Definer anyways, it makes sense to me to just update it to allow injection there.

Reply:
Cool, I haven't totally wrapped my head around the Definers yet.